src/core/public/core_system.test.ts

```diff
@@ -100,16 +100,22 @@ describe('constructor', () => {
 });
 
 describe('#start()', () => {
-  it('calls lifecycleSystem#start() and injectedMetadata#start()', () => {
-    const core = new CoreSystem({
+  let core;
```

Comment:
I thought you were the one in particular that hated this pattern and wanted everyone to use setup functions instead!

Reply:
Haha, I do, but setup functions are best because they take arguments, when there aren't any arguments I'm not super picky. But point taken!

Reply:
There is also the issue that the mocks are cleared between each test, so starting core one and reusing the value for each test wouldn't work anyway.